macros/edr/data_monitoring/anomaly_detection/get_anomaly_scores_query.sql

```diff
@@ -3,7 +3,6 @@
     {%- set full_table_name = elementary.model_node_to_full_name(model_graph_node) %}
     {%- set test_execution_id = elementary.get_test_execution_id() %}
     {%- set test_unique_id = elementary.get_test_unique_id() %}
-
     {% if elementary.is_incremental_model(model_graph_node) %}
       {% set latest_full_refresh = elementary.get_latest_full_refresh(model_graph_node) %}
     {% else %}
```

Comment:
I think we need to add the new fields to the `anomaly_scores_query`.

Reply:
In this file, we select from the `data_monitoring_metrics` and dedup based on the `id` column which is already an md5-hash of the config related fields- so I thought he's enough. Do we want to bring the other columns as a way to externalise them?

Reply:
I think the problem is that if you don't do so, those metrics will still be returned from the `data_monitoring_metrics` CTE, even if they're not supposed to.

Reply:
We also need to take into account the time buckets.